wiki/articles/maven-integration.md

```diff
@@ -1,29 +1,25 @@
 ---
 title: Maven integration
 ---
-Instead of using Gradle to setup your libGDX project, you can also use Maven. However, **this is _not_ recommended!** When issues arise, getting support will most certainly prove very difficult!
```

Comment:
I would leave that as is, the ios integration is seemingly completly broken, the android plugin seems to be not well maintained (last updated 2019) and I don't know about the GWT plugin, according to the docs it is outdated and another plugin should be used.

The archetype that is described later seems to work after some fiddling for at least desktop, but not out of the box.

Reply:
That is fair, maybe the article could mention those details - android and gwt not being maintained, and IOS not being supported currently?

Reply:
> I think something like this at the top of the page would make sense:
> 
> ```
> Setting up your project with Maven is not recommended. Since it is not commonly used in the libGDX community, it may prove difficult to get help if any issues arise!
> {: .notice--info}
> ```

I like that, and I think it could help anyone even more by being more specific. Maven works quite fine. Just because its not commonly used, and only desktop works, doesn't mean people can't have a good experience with it. I don't know why you would keep it so vague.

Reply:
I don't think it's vague. The fact that it is difficult to receive help is IMO the main reason it is not recommended.